packages/db/src/account/account-find-many.ts

```diff
@@ -24,5 +25,5 @@ export async function accountFindMany(db: Database, input: AccountFindManyInput)
     console.log(error)
     throw new Error(`Error finding accounts for wallet id ${parsedInput.walletId}`)
   }
-  return data
+  return data?.map((item) => accountSanitizer(item))
```

Comment:
Do you think we can do this at the middleware layer instead? https://dexie.org/docs/DBCore/DBCore

I think it's possible and likely we forgot to sanitize in future queries we add. Using middleware would remove that concern.

Reply:
@ellipsis-dev create a follow-up issue to investigate Dexie middleware or other ways to make sanitizing work more reliably.

Reply:
Added to the follow-up https://github.com/samui-build/samui-wallet/issues/616#issuecomment-3591586682